test/python/utest_masarclient/test_channelRPC.py

```diff
@@ -0,0 +1,193 @@
+import unittest
+
+from masarclient.channelRPC import ChannelRPC as ChannelRPC
+from masarclient.ntnameValue import NTNameValue
+from masarclient.ntmultiChannel import NTMultiChannel
+from masarclient.nttable import NTTable
+from masarclient.alarm import Alarm
+from masarclient.timeStamp import TimeStamp
+
+'''
+
+Unittests for masarService/python/masarclient/channelRPC.py
+
+'''
+
+
+class TestChannelRPC(unittest.TestCase):
```

Comment:
It looks like all the test methods in this class follow the same basic pattern, perhaps these could be factored out as a helper method?  Hint, as long as the method name doesn't start with 'test' it will be ignored.